### PR TITLE
Juniper compatibility

### PR DESCRIPTION
**Pull request by regisb**
Hi! I noticed that you already migrated this xblock to Python 3.5, so I tried to use it with Tutor. I was happy to discover that I only had to make a few changes, which I submit here.

I did not manage to get this xblock to work with Juniper, yet, but I'm confident that these few changes make sense and should be merged anyway.

# Reviewers

- [ ] who is the person reviewing this changes?
- [ ] perhaps more than one?

# After approval

Keep in mind that after the change have been approved we might still need to do somethings.

- [ ] Squash
- [ ] Bumpversion (major/minor/patch)

### CODE REVIEW

**Comment by morenol** [comment]
Thanks @regisb , we are currently working in the update of this package for Juniper I am going to add here as reviewer the developer that is working in that,

I think that we have to change the way that we use to mock `courseware.model_data` but in anyway I think that this is helpful!

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 *.pyc
-/xblock_flow_control.egg-info/
+/flow_control_xblock.egg-info/
```
Comment (spokerman12) [inline .gitignore:2]: Noted. Thank you for this one!

File: MANIFEST.in [hunk 2]
```diff
@@ -1,3 +1,4 @@
 include README.rst
 include requirements/base.in
 include requirements/test.in
+recursive-include flow_control/static *
```
Comment (spokerman12) [inline MANIFEST.in:4]: Yes! We missed this one.

File: flow_control/flow.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from xblock.fields import Scope, Integer, String
 from xblockutils.studio_editable import StudioEditableXBlockMixin
 from xblock.validation import ValidationMessage
-from courseware.model_data import ScoresClient
+from lms.djangoapps.courseware.model_data import ScoresClient
```
Comment (spokerman12) [inline flow_control/flow.py:13]: You beat me to this change! We'll merge your PR but on this line we'll add a small conditional import so our tests work correctly.

Thank you for your contribution!

**Review by spokerman12 — APPROVED** [review]

**Comment by regisb** [comment]
Thanks for the quick feedback!

**Pull request merged**